src/systemctl/systemctl-show.c

```diff
@@ -1834,6 +1834,26 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
                                 return bus_log_parse_error(r);
 
                         return 1;
+                } else if (contents[0] == SD_BUS_TYPE_STRUCT_BEGIN && streq(name, "OpenFile")) {
+                        char *path, *fdname;
+                        long long options;
+
+                        r = sd_bus_message_enter_container(m, SD_BUS_TYPE_ARRAY, "(ssx)");
+                        if (r < 0)
+                                return bus_log_parse_error(r);
+
+                        while ((r = sd_bus_message_read(m, "(ssx)", &path, &fdname, &options)) > 0) {
+                                bus_print_property_valuef(name, expected_value, flags, "%s:%s:%lld", path, fdname, options);
```

Comment:
You should use "%s:%s:%"PRIu64 as the format string here. That makes sure it's the correct one for both 64-bit and 32-bit compilation